### PR TITLE
Update dataflows-premium-workload-configuration.md

### PR DESCRIPTION
**Pull request by nachoalonsoportillo**
Punctuation and prepended a comment to a sentence whose meaning is not clear to me.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@nachoalonsoportillo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by davidiseminger — COMMENTED** [review]
Thanks for your updates! Online 113, there's a confusing additional word. Its should read:

Then follow the previous steps in this sequence again.

File: powerbi-docs/transform-model/dataflows/dataflows-premium-workload-configuration.md [hunk 1]
```diff
@@ -110,11 +110,11 @@ Slow refresh times are usually a parallelism issue. You should review the follow
 
 5. Do both of the previous steps, allowing for an even higher degree of parallelism, because you get more cooks and a bigger kitchen.
     
-6. If the steps in this section don't provide the desired degree of parallelism, consider upgrading your capacity to a higher SKU. Then follow steps the previous steps in this section again.
+6. If the steps in this section don't provide the desired degree of parallelism, consider upgrading your capacity to a higher SKU. <!--It is not clear what the following sentence means.-->Then follow steps the previous steps in this section again.
```
Comment (v-andreaco) [inline powerbi-docs/transform-model/dataflows/dataflows-premium-workload-configuration.md:113]:
```suggestion
6. If the steps in this section don't provide the desired degree of parallelism, consider upgrading your capacity to a higher SKU. Then follow the previous steps in this sequence again.
```

**Comment by PRMerger18** [comment]
@v-andreaco : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by v-andreaco** [comment]
@davidiseminger 

Is that correct?

**Comment by davidiseminger** [comment]
Yes @v-andreaco  that is correct. Thanks!

**Pull request merged**